src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -180,27 +184,33 @@ public Optional<Phone> getPhone() {
             return Optional.ofNullable(phone);
         }
 
-        public void setEmail(Email email) {
+        public void setEmail(Optional<Email> email) {
             this.email = email;
         }
 
         public Optional<Email> getEmail() {
-            return Optional.ofNullable(email);
```

Comment:
sry i missed ur msg on the tele group, but imo it seems like `ofNullable` is better since it basically does the below 3 lines in 1 cleaner line, but a non-issue really